docs/docs-content/clusters/data-center/nutanix/create-manage-nutanix-cluster.md

```diff
@@ -39,80 +39,92 @@ section guides you in creating a Kubernetes cluster in a Nutanix cloud managed b
 
-8. In the Cluster configuration YAML file that Palette displays, edit the file to replace variables within curly braces
-   listed in the table below with values that apply to your Nutanix cloud environment, and make any adjustments to
-   configure your cluster. Click **Next** when you are done.
+7.  Review profile layers, leaving the OS and Kubernetes packs empty, and customize parameters as desired in the YAML
+    files that display when you select the network and storage layers. Click **Next** when you are done.
 
-| **Variable**                   | **Description**                                                                              |
-| ------------------------------ | -------------------------------------------------------------------------------------------- |
-| `${CLUSTER_NAME}`              | The name of the Nutanix workload cluster. Use the same cluster name you specified in step 5. |
-| `${CONTROL_PLANE_ENDPOINT_IP}` | The Kubernetes API IP endpoint for the cluster you are creating.                             |
-| `${NUTANIX_ENDPOINT}`          | The Nutanix Prism Central IP address.                                                        |
+8.  In the **Cluster Configuration Macros** pane, enter values that apply to your Nutanix cloud environment and, if
+    necessary, adjust the **Cluster configuration** YAML file to configure your cluster. Click **Next** when you are
+    ready to proceed.
 
-:::warning
+    :::info
 
-The following applies when replacing variables within curly braces in the YAML configuration files.
+    The inactive fields are auto-populated from your Nutanix cloud account configuration. When entering other macros,
+    make sure to verify the default values.
 
-    - All the variables must be resolved or have a default value.
+    :::
 
-    - Verify default values such as the port.
+    | **Field**                                      | **Description**                                                                                                                                                                                                                                                                                        |
+    | ---------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+    | **NUTANIX_ADDITIONAL_TRUST_BUNDLE (Optional)** | An environment variable that allows CAPX to verify certificates that are not issued by a publicly trusted certificate authority. For more information, refer to the [Nutanix Certificate Trust](https://opendocs.nutanix.com/capx/v1.2.x/pc_certificates/#configure-an-additional-trust-bundle) guide. |
```

Comment:
We try to avoid adding optional in the field. We prefer a column titled "Required," or we explain it in the description.  But it's also not something we clearly defined.

Reply:
I took this directly from the UI 🤔 It does seem a bit out of the norm, but I wanted to follow the UI to prevent any confusion. Should I remove `(Optional)` and point it out in the description?